pkgs/development/perl-modules/findimagedupes/default.nix

```
@@ -0,0 +1,64 @@
+{ lib, stdenv, fetchurl, makeWrapper, perl, perlPackages, installShellFiles }:
+
+stdenv.mkDerivation rec {
+  pname = "findimagedupes";
+  version = "2.19.1";
+
+  # fetching this from GitHub does not contain the correct version number
+  src = fetchurl {
+    url = "http://www.jhnc.org/findimagedupes/findimagedupes-${version}.tar.gz";
+    sha256 = "sha256-5NBPoXNZays5wzpQYar4uZZb0P/zB7fdecE+SjkJjcI=";
+  };
+
+  # Work around the "unpacker appears to have produced no directories"
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  nativeBuildInputs = [ makeWrapper installShellFiles perlPackages.PodMarkdown ];
+
+  buildInputs = [ perl ] ++ (with perlPackages; [
```

```suggestion
  buildInputs = with perlPackages; [
```
perl should already be in there.

---

without this it throws:
```
/home/user/.nix-profile/bin/findimagedupes: /nix/store/5w6rgxdanp5iszg3rfqgfx9131drxixj-findimagedupes-2.19.1/bin/.findimagedupes-wrapped: /usr/bin/perl: bad interpreter: No such file or directory
/home/user/.nix-profile/bin/findimagedupes: line 3: /nix/store/5w6rgxdanp5iszg3rfqgfx9131drxixj-findimagedupes-2.19.1/bin/.findimagedupes-wrapped: Success
```